spec/services/zendesk_intake_service_spec.rb

```diff
@@ -211,6 +218,13 @@
           data: intake.mixpanel_data.merge(ticket_status.mixpanel_data)
         )
       end
+
+      it "sends a datadog metric" do
```

Comment:
Perhaps a shared examples would DRY this up and also make it easier to add/test other jobs/services in the future?

Reply:
good call. i added a ticket to do that as a follow up.